### ISSUE TITLE
Tiberian Twilight Issue

### ISSUE THREAD

**Issue by ThePlagueDoctor** [Medstar117/CNC-Ultimate-Collection-Fixed-Launchers#9]
Having issues with installing for CNC4 Tiberian Twilight. I get to the point to install it for what games I select yet for some reason TW is greyed out (so is RA, Counterstrike, and The Aftermath but I'm not concerned with those). I have the game installed through origin through the ultimate collection. Any help would be appreciated and I can provide additional info as you request.

**Comment by Medstar117** [comment]
If you've manually moved your game installation elsewhere, then it's likely not able to see your installation. Your game's installation location is stored in the registry, which the installer reads from.

If you did not do this, I would try repairing your game and trying again. Be sure to disable Origin in Game.

**Comment by ThePlagueDoctor** [comment]
Yep, tried those and it's a fresh install so I don't think I've moved it? Do you mean like origin is on C but the game is on a different drive?

**Comment by Medstar117** [comment]
If you installed the game, then manually moved the installation folder somewhere else without telling Origin where it was moved to

That's what I'm referring to. But it sounds like that's not the case for you

Do you have any 3rd party antivirus installed?

**Comment by ThePlagueDoctor** [comment]
Ah, no I hadn't done that. Also fair, I forgot to try seeing if AV was being an ass. Unfortunately, disabling temporarily didn't seem to fix the issue either.

**Comment by Medstar117** [comment]
Which AV is it?

**Comment by ThePlagueDoctor** [comment]
Malwarebytes. I also checked to see if it quarantined anything but it hadn't.

**Comment by Medstar117** [comment]
Very odd...and are you playing on Windows 10/11? Or are you playing through Wine or something?

**Comment by ThePlagueDoctor** [comment]
Windows 10. I'll have to look into what Wine is now haha.

**Comment by Medstar117** [comment]
Wine is an application for Linux that allows Windows applications to run in an emulated Windows environment

Hmm...let me think on that...

**Comment by ThePlagueDoctor** [comment]
Ahhh, fun. Not into Linux currently but probably will be eventually since I work in IT. If you need anything, let me know

**Comment by Medstar117** [comment]
Go into RegEdit and see if this path is valid on your system:
Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\EA Games\Command Conquer 4 Tiberian Twilight

**Comment by ThePlagueDoctor** [comment]
No, it isn't.

**Comment by Medstar117** [comment]
Okay, that's the main key it's looking at

How about if the path has the WOW6432Node part removed?

**Comment by ThePlagueDoctor** [comment]
Nothing under EA Games from that point.

**Comment by Medstar117** [comment]
Is there anything in the EA Games "folder"?

**Comment by ThePlagueDoctor** [comment]
Nope.

**Comment by Medstar117** [comment]
Okay, that's of concern.

Try using Bibber's fixed launchers here and see if they pick up your installation. You can get them here: https://bibber.eu/downloads/cnc-ultimate-collection-launchers/

If this doesn't work, then reply back here

**Comment by ThePlagueDoctor** [comment]
Still nothing. Not sure what is going on.

**Comment by Medstar117** [comment]
Sorry for the delay, got busy IRL. I'll whip you up something as a fix. If you can, we can meet on Discord sometime convenient and you can screen share with me? That way I know exactly what to add.

My handle is Medstar#1550

**Comment by KyleStilkey** [comment]
I'd like to add to this that it very well may be that EA has been making some changes to installers that possibly they updated this as it was recently added/supported under EA Desktop.

I can see that there are registries for all the other games under the Registry location you gave but it looks like the installer it places the registries under a different location. It's being put in Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Electronic Arts\Command and Conquer 4.

The other games exist here too but are instead just the uninstaller info. Not sure why this one is being placed differently but might help fix this problem.

I did move the registries over to the EA Games folder but still no luck with it picking up in the installer.

**Comment by X-Vaskys** [comment]
Don't mean to necro this, but I'm having the same issue:
![image](https://github.com/Medstar117/CNC-Ultimate-Collection-Fixed-Launchers/assets/20264383/f6ff30f6-1b3f-4902-84d9-aa1ce667c8df)

Edit: Bibber's Launchers also cannot recognize the game(s)

**Comment by Medstar117** [comment]
> Don't mean to necro this, but I'm having the same issue: ![image](https://user-images.githubusercontent.com/20264383/252883954-f6ff30f6-1b3f-4902-84d9-aa1ce667c8df.png)
> 
> Edit: Bibber's Launchers also cannot recognize the game(s)

All good, the issue is open so it's fine to comment here 👍 

I have become aware of this now, but there's no real need to worry. The binary that the patcher originally replaced has apparently either been removed or merged with another executable. You shouldn't need to patch Tiberian Twilight for the moment. It also seems that the registry keys that my patcher restored have been fixed as well.

What has also happened is that the registry path for the game has also changed since migrating to the EA App.

TL;DR - You don't need to patch the game. What the patcher originally did has now been fixed by EA. I will make the appropriate changes when I can.
